labs/Casting/readme.md

```diff
@@ -5,7 +5,7 @@ title: Type Casting
 This lab serves multiple goals:
 
 - To help you develop good lab habits,
-- To declare, assign and display variables of different datatypes,
+- To declare, assign and, display variables of different datatypes,
```

Comment:
Really?

Reply:
> Is there a reason that lines 68-79 are commented out?

Yes, it didn't "feel right" to have the students check their results with an IDE, as it could give them a false sense of certainty.